samples/modules/thrift/hello/client/prj.conf

```diff
@@ -1,12 +1,10 @@
 # CONFIG_LIB_CPLUSPLUS Dependencies
-CONFIG_NEWLIB_LIBC=y
-CONFIG_NEWLIB_LIBC_NANO=n
+CONFIG_REQUIRES_FULL_LIBCPP=y
```

Comment:
It might even be better to move this to `select REQUIRES_FULL_LIBCPP` in `modules/thrift/Kconfig`.

Reply:
I wish there were an ISO C code generator upstream (instead of the one that depends on glib)